### ISSUE TITLE
Compatibility issue with latest Phoenix 1.7

### ISSUE THREAD

**Issue by u2ix** [nkezhaya/live_phone#147]
Added this to a app and I receive the following message.
Also tried the suggestion of  adding `{:phoenix_html_helpers, "~> 1.0"}`.  But message doesn't change.

```
==> live_phone
Compiling 3 files (.ex)

== Compilation error in file lib/live_phone.ex ==
** (RuntimeError) use Phoenix.HTML is no longer supported in v4.0.

To keep compatibility with previous versions, add {:phoenix_html_helpers, "~> 1.0"} to your mix.exs deps
and then, instead of "use Phoenix.HTML", you might:

    import Phoenix.HTML
    import Phoenix.HTML.Form
    use PhoenixHTMLHelpers


    (phoenix_html 4.1.1) expanding macro: Phoenix.HTML.__using__/1
    lib/live_phone.ex:8: LivePhone (module)
    (elixir 1.16.2) expanding macro: Kernel.use/1
    lib/live_phone.ex:8: LivePhone (module)
could not compile dependency :live_phone, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile live_phone --force", update it with "mix deps.update live_phone" or clean it with "mix deps.clean live_phone"


```

**Comment by LameckMeshack** [comment]
Do we have a fix for this?

**Comment by nkezhaya** [comment]
I'll have a fix out for this shortly.

**Comment by nkezhaya** [comment]
Fix published with 0.8.0.
